SAMv1.tex

```diff
@@ -197,6 +197,7 @@ \subsection{The header section}
   field is used in the
   alignment records in {\sf RNAME} and {\sf RNEXT} fields. Regular expression: {\tt [!-)+-\char60\char62-\char126][!-\char126]*}\\\cline{2-3}
   & {\tt LN}* & Reference sequence length. \emph{Range}: {\tt [1,2$^{31}$-1]}\\\cline{2-3}
+  & {\tt AH} & Position of an alternate locus (see also \href{http://bit.ly/GRCterm}{http://bit.ly/GRCterm}) in the format of `\emph{chr}:\emph{start}-\emph{end}', `\emph{chr}' or `{\tt *}' (if unknown), where `\emph{chr}' is a contig in the primary assembly. Present \emph{only if} the sequence is an alternate locus.\\\cline{2-3}
```

Comment:
Rob's suggestion here is to say "present only if **this** sequence is an alternate locus", which is a tiny but more explicit change.